Automattic/sensei
# Changed styles for question feedback

Resolves https://github.com/Automattic/sensei/issues/7115

## Proposed Changes
* Updated answer feedback styles for quizzes.

## Testing Instructions


1. Checkout this branch of pro https://github.com/Automattic/sensei-pro/pull/2421
2. Checkout this branch of course theme https://github.com/Automattic/themes/pull/7373
3. Create a course with a lesson with a quiz that has all types of questions
4. Take the quiz as a student, answer some wrong and some right
5. Grade the quiz as a teacher, provide additional feedbacks for some and don't provide for some. Mark some as wrong and some as right.
6. Check the graded quiz from the frontend.
7. Check for all course theme variations and also a couple of top themes apart from course

![Screenshot 2023-09-15 at 9 58 32 PM](https://github.com/Automattic/sensei/assets/6820724/c29795ef-a34c-4d76-8c5f-0a7d678659a5)


## Pre-Merge Checklist
